client/lib/faux-plugins.js

```diff
@@ -156,7 +157,7 @@ module.exports = function(roomName) {
             className={this.props.className}
             kind="youtube"
             autoplay="1"
-            start={Math.max(0, Math.floor(Date.now() / 1000 - this.props.startedAt - clientTimeOffset))}
+            start={Math.max(this.props.youtubeTime , Math.floor(Date.now() / 1000 - this.props.startedAt - clientTimeOffset + this.props.youtubeTime ))}
```

Comment:
How about adding `this.props.youtubeTime` outside of the `Math.max()` expression to make the purpose of the `Math.max` more clear?